### PR TITLE
add default port when custom port not specified

### LINKED ISSUE #67: [BUG] Server not starting: Cannot read property 'port' of undefined
**Describe the bug**
The maint.ts do not default to a port number when a custom port is not specified in medusa-config. This error is specific to the projects created using the cli. 

```
(node:5416) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'port' of undefined
    at bootstrap (<PATH TO PROJECT>\dist\main.js:13:48)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5416) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:5416) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**To Reproduce**
1. Run...
2. See error

**Expected behavior**
The server should have a default port to run the application.

**Screenshots**

**Package version:**
 - 1.6

**Additional context**
Not specified in docs to update config file to add a port number.

### PR DESCRIPTION
**Pull request by octalpixel**
Fixes #67

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,29 @@ npm install medusa-extender
 
 That's it, you are now ready to run your server :rocket:
 
+Note: If you want to run the server on a custom port update medusa-config.js as follows:
+
+```
+module.exports = {
```
Comment (adrien2p) [inline README.md:126]: you could replace with 
```
module.exports = {
  /* ... */
  serverConfig:{
	  port: <PORT_NUMBER>
  }
  /* ... */
};
```

Comment (adrien2p) [reply README.md:126]: you will also need to update the `documentations/01...` file and run npm run build:doc

Comment (octalpixel) [reply README.md:126]: Thanks for the pointer! Updated accordingly

**Comment by adrien2p** [comment]
To be able to merge soon Ive took your proposal in the PR #71

**Pull request closed without merging**